src/usecases/retrieveAverageParticipantsInVisit.js

```diff
@@ -0,0 +1,23 @@
+const retrieveAverageParticipantsInVisit = ({ getDb }) => async (trustId) => {
+  if (!trustId) return { error: "A trustId must be provided." };
+
+  const db = await getDb();
+  const [{ average_participants: averageParticipantsInVisit }] = await db.any(
+    `SELECT AVG(participants) as average_participants
```

Comment:
How precise do we want the result? 2 decimal places ? We can define the precision of the AVG result with https://www.postgresqltutorial.com/postgresql-avg-function/

Reply:
Is the average to a decimal place even useful in this context?